app/models/user.rb

```diff
@@ -50,6 +50,22 @@ def send_activation_email
     UserMailer.account_activation(self).deliver_now
   end
 
+  # Sets the password reset attributes.
+  def create_reset_digest
+    self.reset_token = User.new_token
+    update_attribute(:reset_digest, User.digest(reset_token))
+    update_attribute(:reset_sent_at, Time.zone.now)
```

Comment:
you can use update_attributes 
```suggestion
    update_attributes(reset_sent_at: Time.zone.now, reset_digest: User.digest(reset_token))
```